web/react/components/filtered_user_list.jsx

```diff
@@ -37,10 +44,47 @@ class FilteredUserList extends React.Component {
         }
     }
 
-    handleFilterChange(e) {
-        this.setState({
-            filter: e.target.value
-        });
+    searchProfiles() {
```

Comment:
I was originally thinking of doing this in the parent and just passing the function in but I kind of wanted the callback to show it was loading to the user but I never went through with showing the loading state. I also didn't want to have duplicate code. I'll probably just change it to be an asynchronous call and the change in profiles should propagate through and force a re-render. That should probably fix both our issues.

Can't really do `e.target.value` on a button, that's why I did it with the refs